.github/actions/get-workspaces/action.yml

```diff
@@ -10,6 +16,20 @@ runs:
   using: "composite"
 
   steps:
+  - name: Filter out internal dockers members
+    shell: bash
+    if: ${{ inputs.remove-internal-dockers == 'true' }}
+    run: echo "EXCLUDE_FOLDERS=("omics", "core")" >> ${GITHUB_ENV}
```

Comment:
I think this list should be a configuration file somewhere and not hidden in the action code.